src/index.html

```diff
@@ -3,10 +3,426 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+
+  <body class="page">
+    <header class="header">
+      <a class="header__logo" href="#">
+        <img src="images/logo.png" alt="moyo_logo">
+      </a>
+      <nav>
+        <ul class="header__container">
+          <li class="header__item">
+            <a
+              class="header__link is-active" href="#">
+              apple
+            </a>
```

Comment:
fix formatting everywhere
```suggestion
            <a class="header__link is-active" href="#">
              apple
            </a>
```